SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -186,7 +190,64 @@ public class SwiftyTesseract {
     completionHandler(swiftString)
     
   }
-
+  
+  /// Takes an array UIImages and returns the PDF as a data object
+  ///
+  /// - Parameter images: Array of UIImages that should be analyzed
+  /// - Returns: PDF as Data object
+  /// - Throws: SwiftyTesseractError in case something went wrong
+  public func createPDF(from images: [UIImage]) throws -> Data {
+    let _ = semaphore.wait(timeout: .distantFuture)
+    
+    // create unique file path
+    let file = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(NSUUID().uuidString)
+    
+    guard let renderer = TessPDFRendererCreate(file.path, bundle.pathToTrainedData, 0) else {
+        throw SwiftyTesseractError.unableToCreateRenderer
+    }
+    
+    guard TessResultRendererBeginDocument(renderer, "Unkown Title") == 1 else {
+      TessDeleteResultRenderer(renderer)
+      throw SwiftyTesseractError.unableToBeginDocument
+    }
+    
+    var pixImages = [Pix]()
```

Comment:
Instead of creating an empty array and then iterating over the array passed in as an argument, you could just map the `[UIImage]` passed in:
```suggestion
let pixImages = images.map(createPix)
```